StellarApp/Controllers/ContainerViewController.swift

```diff
@@ -1,30 +1,21 @@
-//
-//  ContainerViewController.swift
-//  StellarApp
-//
-//  Created by TingxinLi on 4/8/19.
-//  Copyright © 2019 TingxinLi. All rights reserved.
-//
-
 import UIKit
 
 class ContainerViewController: UIViewController {
```

Comment:
Is this VC used anywhere? If not I think it's safe to delete it.

Reply:
yes, we will need to working on this later